src/funding/paypal/template.jsx

```diff
@@ -147,10 +147,60 @@ function ButtonPersonalization(opts : LabelOptions) : ?ChildType {
 }
 
 
+const getExperimentOneAnimation = (opts) => {
+    if (!opts.enableExperimentOne) {
+        return;
+    }
+    const experimentStyles = (<style innerHTML={ `
+        .${ CLASS.DOM_READY } .${ LOGO_CLASS.LOGO }{
+            animation: 1s experiment-one-move-logo-to-the-left 2s forwards;
+            position: relative;
+        }
+
+        .${ EXPERIMENT_ONE.PLACE_HOLDER } {
+            display: inline-block;
+            position: absolute;
+            opacity: 0; 
+            animation: 1s experiment-one-move-placeholder-to-the-right 2s forwards;
+        }
+
+        .${ CLASS.BUTTON_LABEL } .${ EXPERIMENT_ONE.PLACE_HOLDER } span {
+            font-size: 14px;
+            color: #142C8E;
+            padding-top: 3px;
+            font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+        }
+
+        @keyframes experiment-one-move-logo-to-the-left {
+            100% {
+                transform: translateX(-300px)
+            }
+        }
+        
+        @keyframes experiment-one-move-placeholder-to-the-right {
+            0%{
+                opacity: 1;
+            }
+            100% {
+                transform: translateX(220px);
+                opacity: 1;
+            }
+        }
+    ` }
+    />);
+    return (
+        <Fragment>
+            <div class="experiment-one-label"> <span>Earn rewards</span></div>
```

Comment:
Are there localization requirements for "Earn rewards"?

Reply:
this label will come from a machine learning model prediction, but for now will be harcoded